### ISSUE TITLE
Support zeus-0.14.0.rc1 and later

### ISSUE THREAD

**Issue by morganchristiansson** [sevos/zeus-parallel_tests#23]
The latest version of zeus is 0.14.0.rc1, however it's not compatible with zeus-parallel_tests

```
Bundler could not find compatible versions for gem "zeus":
  In Gemfile:
    zeus-parallel_tests (>= 0) ruby depends on
      zeus (~> 0.13.3) ruby

    zeus (0.14.0.rc1)
```

(I'm trying to use zeus-parallel_tests with a rails engine and it's not working. I thought latest version of zeus might help - but it's not compatible with this gem.)

**Comment by schneidmaster** [comment]
I am (as of a few days ago) the new maintainer for this gem, and I'm looking through the old issues now. I am actively working on updating the gem for compatibility with zeus 0.15.4. I will close this issue when that is done - hopefully within a week or two depending on my schedule.

**Comment by schneidmaster** [comment]
Fixed by #37. This has not been deployed to rubygems yet but I will comment again in this issue when it has.
